package.json

```diff
@@ -22,6 +22,7 @@
     "lint-prune": "! ts-prune | grep -v \"used in module\"",
     "clean": "find . -name node_modules -o -name .turbo -o -name .next -o -name dist -type d -prune | xargs rm -rf",
     "codegen:override-prisma": "tsx scripts/addPrismaOverrides.ts",
+    "codegen:entrypoints": "turbo --filter=\"@trpc/*\" codegen:entrypoints",
```

Comment:
I am stupid and don't understand turbo completely, but in order to trigger this I had to do `p codegen:entrypoints -- no-cache`.

Not sure if this should even be here or if it should just be magically triggered 😅

Reply:
it should be triggered automatically on build

Reply:
maybe package.json should be in the outputs too actually